source/reference/hyperlinks.txt

```diff
@@ -0,0 +1,133 @@
+.. _style-guide-hyperlinks:
+
+==========
+Hyperlinks
+==========
+
+You can link out to other sites, internal pages, or specific sections on
+a page using inline hyperlinks and cross-referenced hyperlinks.
+
+Inline Hyperlinks
+-----------------
+
+Use inline hyperlinks when you want to link to a website with its full URL.
+
+Example
+~~~~~~~
+
+.. code-block:: rst
+
+   Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Rendered View
+`````````````
+
+Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Cross-Referenced Hyperlinks
+---------------------------
+
+Use cross-referenced hyperlinks when you want to link to another page in
+docs. This page can be in the same repo, or another within the MongoDB
+organization.
+
+Procedure
+~~~~~~~~~
+
+.. procedure::
+   :style:  normal
+
+   .. step:: Check if the cross-referenced hyperlink you want exists.
+
+      A cross-referenced hyperlink has the following format:
+
+      .. code-block:: rst
+
+         .. _<some-text>:
+
+      If the cross-referenced hyperlink is in a different MongoDB repo, 
+      add the documentation URL landing page and append
+      ``/objects.inv``. Then, add the URL to the  ``intersphinx`` array
+      in the ``snooty.toml`` file and skip the next step. See the
+      :ref:`different-repo-example` for an example.
+
+      If someone else already created the cross-referenced hyperlink in
+      the same repo, skip the next step.
+   
+   .. step:: Create the cross-referenced hyperlink.
+      
+      Place your cross-referenced hyperlink a line before where you 
+      want the reference to be.
+
+      .. note::
+
+         You need to place this two lines before where you want the link
+         to. See the :ref:`same-repo-example` for an example.
+
+   .. step:: Reference the cross-referenced hyperlink using ``:ref:``.
```

Comment:
We should probably link to the cross references page (that I'm working on now) somewhere around here. The ref is .. _style-guide-cross-references:

Reply:
I'll add a TODO link type of things just because your page isn't merged in yet